pem/lib/pem.dart

```diff
@@ -111,6 +111,12 @@ enum PemLabel {
   /// See [section 10 RFC 7468](https://tools.ietf.org/html/rfc7468#section-10).
   privateKey,
 
+  /// Unencrypted PKCS #8 Private Key Information Syntax structure
+  /// (`'OPENSSH PRIVATE KEY'`).
+  ///
+  /// See [section 10 RFC 7468](https://tools.ietf.org/html/rfc7468#section-10).
```

Comment:
I don't see anything mentioning `"OPENSSH PRIVATE KEY"` in:
https://datatracker.ietf.org/doc/html/rfc7468#section-10

`unsafeIgnoreLabel` exists for things outside the spec.